src/results-output/results-doc.ts

```diff
@@ -266,10 +266,13 @@ export function appendCurrentTopLevelForm() {
   void appendFormGrabbingSessionAndNS(true);
 }
 
-function lastLineIsEmpty(doc: vscode.TextDocument): boolean {
-  const { lineCount } = doc;
-  const lastLine = doc.getText(new vscode.Range(lineCount - 1, 0, lineCount - 1, Infinity));
-  return lastLine === '';
+export async function lastLineIsEmpty(): Promise<boolean> {
+  try {
+    const doc = await vscode.workspace.openTextDocument(DOC_URI());
+    return util.lastLineIsEmpty(doc);
+  } catch (error) {
+    console.error('Failed opening results doc', error);
```

Comment:
Technically I think this caught error could also be from `util.lastLineIsEmpty`, however unlikely that might be. The return and call to that could be moved out of the try/catch to avoid that, but it's not really that important - just noting here.